src/main/java/com/monitorjbl/xlsx/impl/StreamingRow.java

```diff
@@ -85,6 +85,13 @@ public Cell createCell(int column, int type) {
   }
 
   /**
+   * Not supported.  Added for compatability with POI 3.12.
+   */
+  public int getOutlineLevel() {
```

Comment:
as indicated in the JavaDoc, I am using POI 3.12 (the latest stable version).  The code will not compile without implementing this Cell method (https://poi.apache.org/apidocs/org/apache/poi/ss/usermodel/Cell.html#removeHyperlink()).